zgw/zaakdetails/src/main/kotlin/com/ritense/zaakdetails/domain/ZaakdetailsObject.kt

```diff
@@ -0,0 +1,22 @@
+package com.ritense.zaakdetails.domain
+
+import jakarta.persistence.Column
+import jakarta.persistence.Entity
+import jakarta.persistence.Id
+import jakarta.persistence.Table
+import java.net.URI
+import java.util.UUID
+
+@Entity
+@Table(name = "zaakdetails_object")
+data class ZaakdetailsObject(
+    @Id
+    @Column(name = "id")
```

Comment:
This has to be the document url based on the way it is used. I would like to see this reflected in the name as well.